### ISSUE TITLE
psx mini classic controller support?

### ISSUE THREAD

**Issue by spielvan** [xjsxjs197/WiiSXRX_2022#84]
psx mini classic controller support?

**Comment by carnage702** [comment]
that would require the emulator to support usb hid controllers which it doesnt no wii homebrew app supports hid controllers besides nintendont/devolution afaik.

**Comment by carnage702** [comment]
> > that would require the emulator to support usb hid controllers which it doesnt no wii homebrew app supports hid controllers besides nintendont/devolution afaik.
> 
> Can't a implementation based on Nintendont be made?

if it was easy to implement hid controls then all wii apps would have it, there is a reason none do, besides those 2.

**Comment by xjsxjs197** [comment]
The work of supporting HID controllers is almost completed. If you want to add controllers beyond what nintendont supports, you need to add a new INI configuration file. The process should be as follows:
Connect your HID controller to the Wii, and then start HID_Test.dol (please modify it to boot. dol). If the controller is supported, you will see normal connection information and information when each button is pressed. 
Send me the information when each button is pressed, and I will create an INI file. 
You can refer to the information in this link:
https://gbatemp.net/threads/nintendont-custom-controller-configuration-file.633068/#post-10164777

[HID_Test.zip](https://github.com/xjsxjs197/WiiSXRX_2022/files/14756167/HID_Test.zip)

**Comment by Invictaz** [comment]
> that would require the emulator to support usb hid controllers which it doesnt no wii homebrew app supports hid controllers besides nintendont/devolution afaik.

Not true. Fakemote does support HID controllers. All it needs is an adaptation to use the PSX mini controller.

**Comment by carnage702** [comment]
> > that would require the emulator to support usb hid controllers which it doesnt no wii homebrew app supports hid controllers besides nintendont/devolution afaik.
> 
> Not true. Fakemote does support HID controllers. All it needs is an adaptation to use the PSX mini controller. @xjsxjs197 why is the Nintendont approach preferred with a lot of seperate .ini files while Fakemote does all that is needed?

you can hardcode hid controllers directly on any emulator, but ppl prefer the files to make their own configurations imo.
